keeper/keeper.go

```diff
@@ -55,7 +56,7 @@ func NewKeeper(
 
 // Logger returns the application logger, scoped to the associated module
 func (k Keeper) Logger(ctx sdk.Context) log.Logger {
-	return ctx.Logger().With("module", fmt.Sprintf("x/%s-%s", host.ModuleName, types.ModuleName))
+	return ctx.Logger().With("module", fmt.Sprintf("x/%s-%s", ibcexported.ModuleName, types.ModuleName))
```

Comment:
I know you didn't write this.. but since we're here, do we need this first string? Could it just be `x/interchainquery` ?

Reply:
Dis-regard. I see it says "scoped to the associated module".